spec/property.dd

```diff
@@ -283,6 +283,30 @@ $(D object.TypeInfo_Class)).)
 $(P $(CODE .classinfo) applied to an interface gives the information for the
 interface, not the class it might be an instance of.)
 
+$(H2 $(LNAME2 tupleof, .tupleof Property))
+
+$(P $(CODE .tupleof) provides a tuple representation of a struct. That way,
+it is possible to iterate on each by using $(CODE foreach).)
```

Comment:
```suggestion
$(P $(CODE .tupleof) provides a symbol sequence of all the non-static fields in a struct or class.
The order of the fields in the tuple matches the order in which the fields are declared.

For classes, hidden fields and fields of the bass class are excluded.

A common use case is to iterate over the fields with a $(CODE foreach) loop.)
```

Reply:
Thanks for the input!

Reply:
IMHO the mention of the `foreach` use-case is a bit specific for a language spec. (One could also access the sequence members using indexes.)

Reply:
See also: <https://dlang.org/spec/class.html#tupleof>